content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -1636,17 +1636,17 @@ Used to restart an archived chat.
 
 <Text>
 
-### Close Thread
+### Deactivate Chat
 
 Closes the thread. Sending messages to this thread will no longer be possible.
```

Comment:
Perhaps, we should adjust the method a little bit and refer to its name?
Something like:
**"Deactivates a chat by closing the currently open/active thread."**

Reply:
open/active is for us to decide, I don't want to add both to the definition

Reply:
Sounds great. At first, I left it intentionally, because I wanted to have an info about closing a thread. But your proposition satisfies both cases.